src/extension/services/validationService/checks/macos.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as nls from "vscode-nls";
+import {
+    basicCheck,
+    createNotFoundMessage,
+    createVersionErrorMessage,
+    parseVersion,
+} from "../util"; // executeCommand
```

Comment:
Looks like this comment is redundant.

```suggestion
} from "../util";
```